features/prompt_memes_b/challenge_promptmeme_posting_fills.feature

```diff
@@ -458,3 +459,41 @@ Feature: Prompt Meme Challenge
   Then I should see "In response to a prompt by myname4"
     And I should see "Fandom: Stargate Atlantis"
     And I should see "Anonymous" within ".byline"
+    And I should see a link "prompt"
+
+  Scenario: Fic links to the prompt it fulfils, for all users
+
+  Given I have Battle 12 prompt meme fully set up
+  When I am logged in as "myname1"
+  When I sign up for Battle 12 with combination B
+    And I am logged in as "myname4"
+    And I claim a prompt from "Battle 12"
+  When I fulfill my claim
+  When I reveal works for "Battle 12"
+  When I view the work "Fulfilled Story"
+  Then I should see "Fulfilled Story"
+    And I should see "In response to a prompt by Anonymous"
+    And I should see a link "prompt"
+    And I follow "prompt"
+    And I should see "Request by Anonymous"
```

Comment:
When we're switching from checking the results (all of those "I should see" steps) to taking an action, we like to switch back to "When" -- and then back to "Then" for checking the results. So:

```
When I follow "prompt"
Then I should see "Request by Anonymous"
```

Lines 484 & 485, 491 & 492, and 498 & 499 also need this change.